docs/kqueen.rst

```diff
@@ -364,43 +364,43 @@ You can provision a Kubernetes cluster using various community of engines, such
 To provision a Kubernetes cluster using the Google Kubernetes Engine:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Login in to the Google Kubernetes Engine (https://console.cloud.google.com).
-2. Select your Project.
-3. Navigate to the ``API’s & Services``  -> ``Credentials`` tab and click ``Create credentials``.
-4. From ``Service Account key``, select your service account.
-5. Select Json as the key format.
-6. Download the Json snippet.
-7. Log in to the KQueen web UI.
-8. From the ``Create Provisioner`` tab, select ``Google Kubernetes Engine``.
-9. Insert the downloaded Json snippet that contains the service account key and submit the provisioner creation.
-10. Click ``Deploy Cluster``.
-11. Select the defined GCE provisioner.
-12. Specify the cluster requirements.
-13. Click ``Submit``.
-14. To track the cluster status, navigate to the KQueen main dashboard.
+#. Login in to the Google Kubernetes Engine (https://console.cloud.google.com).
+#. Select your Project.
+#. Navigate to the ``API’s & Services``  -> ``Credentials`` tab and click ``Create credentials``.
+#. From ``Service Account key``, select your service account.
+#. Select Json as the key format.
+#. Download the Json snippet.
+#. Log in to the KQueen web UI.
+#. From the ``Create Provisioner`` tab, select ``Google Kubernetes Engine``.
+#. Insert the downloaded Json snippet that contains the service account key and submit the provisioner creation.
```

Comment:
#. Click ``Create Provisioner``.
#. Select ``GKE`` and insert the downloaded Json snippet that contains the service account key.
#. Click ``Submit``.